.github/CODEOWNERS

```diff
@@ -536,6 +536,7 @@
 /jinja_utils*.py @jameesjohn @vojtechjelinek
 /webpack.common.config.ts @jameesjohn @vojtechjelinek
 /webpack.common.macros.ts @jameesjohn @vojtechjelinek
+/webpack.terser.config.ts @nishantwrp @vojtechjelinek
```

Comment:
Move this after `/webpack.prod.config.ts`

Reply:
Done!